### PR TITLE
Use images for admin icons

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/assets/stylesheets/admin/base.scss [hunk 1]
```diff
@@ -1306,3 +1306,152 @@ table .ellipsis {
...
+  height: 12px;
+  background-image: image-url("icon-valid.png");
+  background-size: 15px 12px;
+
+  @include is-retina {
+    background-image: image-url("icon-valid@2.png");
+  }
+}
+
+#password-result-span,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid using id selectors

File: app/assets/stylesheets/admin/base.scss [hunk 2]
```diff
@@ -1306,3 +1306,152 @@ table .ellipsis {
...
+#password-confirmation-result-span {
+  .glyphicon-ok {
+    opacity: 0.25;
+    -ms-filter:"progid:DXImageTransform.Microsoft.Alpha(Opacity=25)";
+    filter: alpha(opacity=25);
+  }
+}
+
+.js .good-password #password-result-span,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid using id selectors

File: app/assets/stylesheets/admin/base.scss [hunk 3]
```diff
@@ -1306,3 +1306,152 @@ table .ellipsis {
...
+  height: 13px;
+  background-image: image-url("icon-edit.png");
+  background-size: 15px 13px;
+
+  @include is-retina {
+    background-image: image-url("icon-edit@2.png");
+  }
+}
+
+.list-actions .glyphicon-pencil {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Merge rule `.list-actions .glyphicon-pencil` with rule on line 330

File: app/assets/stylesheets/admin/base.scss [hunk 4]
```diff
@@ -1306,3 +1306,152 @@ table .ellipsis {
...
+.js .good-password #password-result-span,
+.js .good-password #password-confirmation-result-span.confirmation-matching {
+  .glyphicon-ok {
+    opacity: 1;
+    -ms-filter:"progid:DXImageTransform.Microsoft.Alpha(Opacity=100)";
+    filter: alpha(opacity=100);
+  }
+}
+
+.glyphicon-remove {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Merge rule `.glyphicon-remove` with rule on line 1400

File: app/assets/stylesheets/admin/base.scss [hunk 5]
```diff
@@ -1306,3 +1306,152 @@ table .ellipsis {
...
+  @include is-retina {
+    background-image: image-url("icon-valid@2.png");
+  }
+}
+
+#password-result-span,
+#password-confirmation-result-span {
+  .glyphicon-ok {
+    opacity: 0.25;
+    -ms-filter:"progid:DXImageTransform.Microsoft.Alpha(Opacity=25)";
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Colon after property should be followed by one space<br>Avoid vendor prefixes.

File: app/assets/stylesheets/admin/base.scss [hunk 6]
```diff
@@ -1306,3 +1306,152 @@ table .ellipsis {
...
+    filter: alpha(opacity=25);
+  }
+}
+
+.js .good-password #password-result-span,
+.js .good-password #password-confirmation-result-span.confirmation-matching {
+  .glyphicon-ok {
+    opacity: 1;
+    -ms-filter:"progid:DXImageTransform.Microsoft.Alpha(Opacity=100)";
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Colon after property should be followed by one space<br>Avoid vendor prefixes.

File: app/assets/stylesheets/admin/base.scss (hunk 1 above)
Comment (hammerfunk) [inline app/assets/stylesheets/admin/base.scss]: ignore

File: app/assets/stylesheets/admin/base.scss [hunk 7]
```diff
@@ -1306,3 +1306,148 @@ table .ellipsis {
...
+  width: 15px;
+  height: 12px;
+  background-image: image-url("icon-valid.png");
+  background-size: 15px 12px;
+
+  @include is-retina {
+    background-image: image-url("icon-valid@2.png");
+  }
+
+  #password-result-span &,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid using id selectors

File: app/assets/stylesheets/admin/base.scss [hunk 8]
```diff
@@ -1306,3 +1306,148 @@ table .ellipsis {
...
+  #password-confirmation-result-span & {
+    opacity: 0.25;
+    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=25)";
+    filter: alpha(opacity=25);
+  }
+
+  .js .good-password #password-result-span &,
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid using id selectors

File: app/assets/stylesheets/admin/base.scss [hunk 9]
```diff
@@ -1306,3 +1306,148 @@ table .ellipsis {
...
+  background-size: 15px 12px;
+
+  @include is-retina {
+    background-image: image-url("icon-valid@2.png");
+  }
+
+  #password-result-span &,
+  #password-confirmation-result-span & {
+    opacity: 0.25;
+    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=25)";
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid vendor prefixes.

File: app/assets/stylesheets/admin/base.scss [hunk 10]
```diff
@@ -1306,3 +1306,148 @@ table .ellipsis {
...
+    filter: alpha(opacity=25);
+  }
+
+  .js .good-password #password-result-span &,
+  .js .good-password #password-confirmation-result-span.confirmation-matching & {
+    opacity: 1;
+    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=100)";
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid vendor prefixes.

File: app/assets/stylesheets/admin/base.scss (hunk 4 above)
Comment (hammerfunk) [inline app/assets/stylesheets/admin/base.scss]: Done

**Comment by dreamfall** [comment]
looks good :+1:

**Comment by hammerfunk** [comment]
Non related spec errors

**Pull request merged**